webdriver/tests/bidi/input/perform_actions/pointer_origin.py

```diff
@@ -38,40 +36,43 @@ async def test_viewport_inside(bidi_session, top_context, get_actions_origin_pag
 
     actions = Actions()
     actions.add_pointer().pointer_move(x=point["x"], y=point["y"])
-    await bidi_session.input.perform_actions(
-        actions=actions, context=top_context["context"]
-    )
+    await bidi_session.input.perform_actions(actions=actions,
+                                             context=top_context["context"])
 
-    click_coords = await get_click_coordinates(bidi_session, context=top_context)
+    click_coords = await get_click_coordinates(bidi_session,
+                                               context=top_context)
```

Comment:
What's the difference to line 53 for being formatted differently?

Reply:
Because it exceed 80 lines